outputs.tf

```diff
@@ -19,21 +19,21 @@ output "eni_flow_ids" {
 }
 
 output "kinesis_id" {
-  value       = "${aws_kinesis_stream.default.id}"
+  value       = "${aws_kinesis_stream.default.*.id}"
```

Comment:
```suggestion
  value       = "${join("", aws_kinesis_stream.default.*.id)}"
```

Reply:
@aknysh did you intend to not apply the join+splat pattern to the aws_flow_log.subnets and aws_flow_log.eni outputs as well?